src/ua/dymohlo/geometricshapes/Square.java

```diff
@@ -0,0 +1,15 @@
+package ua.dymohlo.geometricshapes;
+
+public class Square extends Figure {
+    double squareSide = 5;
```

Comment:
Please, use constructor to initialize variables
For the record: class fields should be private